src/swagger/auth.ts

```diff
@@ -1,15 +1,18 @@
-export const getAuthentication = (type?: 'basic') => {
-  switch (type) {
-    case 'basic':
-      return {
-        securityDefinitions: {
-          basicAuth: {
-            type: 'basic',
-          },
-        },
-        security: [{ basicAuth: [] }],
-      };
-    default:
-      return {};
-  }
+enum AuthenticationType {
+  BASIC = 'basic',
+}
+
+const basicConfig = {
+  securityDefinitions: {
+    basicAuth: {
+      type: 'basic',
+    },
+  },
+  security: [{ basicAuth: [] }],
 };
+
+const authenticationConfiguration = {
+  [AuthenticationType.BASIC]: basicConfig,
+};
+
+export const getAuthentication = (type?: 'basic') => authenticationConfiguration[type];
```

Comment:
could remove the need for a method call here by using an enum

Reply:
That is very much correct sir